admin/view/wp-slimstat-reports.php

```diff
@@ -1741,11 +1637,11 @@ public static function show_world_map()
                                 }
                                 ?>
                             </div>
-                            <strong><?= esc_html($country['name']) ?></strong>
+                            <strong><?php echo esc_html($country['name']) ?></strong>
                             <div class="bar-container">
-                                <div class="bar-fill" style="width: <?= $country['percent'] ?>%;"></div>
+                                <div class="bar-fill" style="width: <?php echo $country['percent'] ?>%;"></div>
                             </div>
-                            <span><?= $country['percent'] ?>%</span>
+                            <span><?php echo $country['percent'] ?>%</span>
```

Comment:
Let's make sure we escape everything. also adding let's mke sure we ending the PHP codes with `;`.